<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mynotpadClass</class>
 <widget class="QMainWindow" name="mynotpadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mynotpad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="action_A"/>
    <addaction name="action_P"/>
    <addaction name="action_E"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newAction">
   <property name="text">
    <string>新建（&amp;N）</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开（&amp;O）</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存（&amp;S）</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>另存为（&amp;A）</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>打印（&amp;P）</string>
   </property>
  </action>
  <action name="action_E">
   <property name="text">
    <string>退出（&amp;E）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mynotpad.qrc"/>
 </resources>
 <connections/>
</ui>
